### PR TITLE
[WIP] Pricing page

### PR DESCRIPTION
**Pull request by dlapiduz**
This is what we started with @ozzyjohnson the other day.

@mogul this is still WIP but we would like to add the stuff we are doing in the spreadsheet here.

### CODE REVIEW

File: pages/pricing.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Network security
+
+
+## How much does it cost?
+
+Support tiers:
+- FISMA Low
+  $60k/yr for 3 "systems"
+- FISMA Mod
+  $90k/yr for 1 "system"
```
Comment (msecret) [inline pages/pricing.md:27]: Why would somebody pay more for less systems?

Comment (dlapiduz) [inline pages/pricing.md:27]: FISMA Moderate requires more "support" in our opinion. Basically a lot more support :)

Comment (dlapiduz) [inline pages/pricing.md:27]: I am not sure if this makes a huge amount of sense though

Comment (msecret) [inline pages/pricing.md:27]: It did seem confusing to me.

**Comment by afeld** [comment]
Closes #20.

**Comment by afeld** [comment]
Would it make sense for this page to be in docs.cloud.gov? Seems that's where we put all our other detail pages, though I would see the argument for having it be pushed out to the landing page.

**Comment by mogul** [comment]
I feel the same although I think realistically we should just be thinking about putting cg-landing and cg-docs in the same repository.

**Comment by mogul** [comment]
Will take this up in cg-docs.

**Comment by dlapiduz** [comment]
:clap: on merging this and cg-docs!

**Comment by afeld** [comment]
https://trello.com/c/EsE2edi0/287-publish-a-one-page-pricing-model-for-feedback

**Comment by afeld** [comment]
https://trello.com/c/5SSgckP6/577-merge-the-landing-and-docs-sites

**Comment by afeld** [comment]
Moved to https://github.com/18F/cg-docs/pull/160.

**Pull request closed without merging**